kernel/packages/shared/dao/sagas.ts

```diff
@@ -41,12 +41,51 @@ const CATALYST_CANDIDATES_KEY = CACHE_KEY + '-' + SET_CATALYST_CANDIDATES
 const CACHE_TLD_KEY = 'tld'
 
 export function* daoSaga(): any {
+  yield call(loadDefaultCatalystRealms)
   yield takeEvery(WEB3_INITIALIZED, loadCatalystRealms)
 
   yield takeEvery([INIT_CATALYST_REALM, SET_CATALYST_REALM], cacheCatalystRealm)
   yield takeEvery([SET_CATALYST_CANDIDATES, SET_ADDED_CATALYST_CANDIDATES], cacheCatalystCandidates)
 }
 
+function* loadDefaultCatalystRealms() {
```

Comment:
I suppose we need this to be able to run the avatar editor, right? isn't there another workaround?

Reply:
It's necessary to kwon if a profile already exists or not. Today the realm is initialized after calling `loginCompleted` but now we need to kwon if the address has a profile before `loginCompleted` to kwon if user need to continue with the sign up or sing in flow.